.nvmrc

```diff
@@ -0,0 +1 @@
+21.6.2

```

Comment:
We should add the Long-Term Support (LTS) version of Node.js rather than the current/latest version.
You should also update the package.json and include the "engine"